src/App.tsx

```diff
@@ -15,71 +15,80 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const goodsList = goods.map(good => (
+    <tr
+      data-cy="Good"
+      key={good}
+      className={
+        (selectedGood === good)
+          ? ('has-background-success-light')
+          : ''
+      }
```

Comment:
[classnames](https://www.npmjs.com/package/classnames)

Reply:
please contact me to explain this one in more detail, it has been installed.

Reply:
https://www.npmjs.com/package/classnames#usage